File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    answer = 0
    o = ''
    e = ''
    for n in num_list:
        if n%2 ==0:
            e += str(n)
        else:
            o += str(n)
    answer = int(e) + int(o)
    return answer

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    let odd = [];
    let even = [];
    for(let num of num_list){
        if(num%2==0){
            even.push(num);
        }else{
            odd.push(num);
        }
    }
    let e_num = parseInt(even.join(''));
    let o_num = parseInt(odd.join(''));
    
    return e_num+o_num;
}